packages/cli/src/commands/init/init.ts

```diff
@@ -383,6 +394,9 @@ export default (async function initialize(
     projectName = projName;
   }
 
+  // if the project with the name already has cache, remove the cache to avoid problems with pods installation
+  cacheManager.removeProjectCache(projectName);
```

Comment:
Hm, the same scenario could be if someone clones repo?

Reply:
yeah, but is there any way to prevent that in this scenario? 🤔